mica/stats/acq_stats.py

```diff
@@ -542,17 +548,45 @@ def calc_stats(obsid):
     # Filter the catalog to be just acquisition stars
     catalog = catalog[(catalog['type'] == 'ACQ') | (catalog['type'] == 'BOT')]
     time = DateTime(guide_start).secs
+    dc_max = get_dark_map_bright(time, catalog)
     ccd_temp = np.mean(fetch_sci.MSID('AACCCDPT',
                                       time-250,
                                       time+250).vals)
     warm_threshold = 100.0
     warm_frac = mica.archive.aca_dark.dark_model.get_warm_fracs(
         warm_threshold, time, ccd_temp)
     temps = {'ccd_temp': ccd_temp, 'n100_warm_frac': warm_frac}
-    return obsid_info, acq_stats, star_info, catalog, temps
-
-
-def table_acq_stats(obsid_info, acq_stats, star_info, catalog, temp):
+    return obsid_info, acq_stats, star_info, catalog, temps, dc_max
+
+
+def get_dark_map_bright(time, catalog):
+    dark_cal = mica.archive.aca_dark.get_dark_cal_props(time, select='before', include_image=True)
+    image = dark_cal['image']
+    row_idx = np.repeat(np.arange(-512, 512), 1024).reshape(1024,1024)
+    col_idx = np.repeat(np.arange(-512, 512), 1024).reshape(1024,1024).transpose()
+    slot_worst_pix = {}
+    for slot in catalog:
+        corner1 = yagzag_to_pixels(slot['yang'] - slot['halfw'], slot['zang'] + slot['halfw'])
+        corner2 = yagzag_to_pixels(slot['yang'] + slot['halfw'], slot['zang'] - slot['halfw'])
+        row_ranges = (int(corner2[0]) - 1, int(np.ceil(corner1[0])) + 1)
```

Comment:
This could use a comment indicating I'm explicitly padding by a pixel after rounding up or down.